draft-ietf-quic-v2.md

```diff
@@ -145,17 +145,18 @@ with clients. In particular, HTTP clients often use Alt-Svc {{?RFC7838}} to
 discover QUIC support. As this mechanism does not currently distinguish between
 QUIC versions, HTTP servers that support multiple versions reduce the
 probability of incompatibility and the cost associated with QUIC version
-negotiation or TCP fallback. For example, an origin advertising support for "h3" in Alt-Svc
-SHOULD support QUIC version 1 as it was the original QUIC version used by HTTP/3 and
-therefore some clients will only support that version.
+negotiation or TCP fallback. For example, an origin advertising support for "h3"
+in Alt-Svc SHOULD support QUIC version 1 as it was the original QUIC version
+used by HTTP/3 and therefore some clients will only support that version.
 
-Any QUIC endpoint that supports multiple versions MUST meet the minimum
+Any QUIC endpoint that supports QUIC version 2 MUST meet the minimum
```

Comment:
"minimum requirements" isn't well-defined in the VN draft. I'd suggest being more specific and requiring
* send the TP
* receive the TP
* validate the received TP